feature-libs/user/profile/components/register/register-component.service.ts

```diff
@@ -18,7 +20,8 @@ import { Observable } from 'rxjs';
 export class RegisterComponentService {
   constructor(
     protected userRegisterFacade: UserRegisterFacade,
-    protected globalMessageService: GlobalMessageService
+    protected globalMessageService: GlobalMessageService,
+    protected fb: UntypedFormBuilder
```

Comment:
New required constructor param is a breaking change, not allowed in minor. 

Let's make it optional:

```diff
- protected fb: UntypedFormBuilder
+ protected fb?: UntypedFormBuilder
```

Reply:
done